app/form-components/team-form/TeamMembersForm.tsx

```diff
@@ -61,6 +66,16 @@ interface Props {
   currentUser: LoggedInUser | null;
 }
 
+export const isValidGovEmail = (email: string) => {
```

Comment:
What about we use the library to check the validity of the email? e.g.
https://github.com/bcgov/sso-requests/blob/dev/app/layout/UserProfileModal.tsx#L66

so the logic would be `validator.isEmail(email) && email.endsWith('@gov.bc.ca')`;
and won't need the `try catch` block for this function.